src/style.css

```
@@ -0,0 +1,121 @@
+/* stylelint-disable color-hex-length */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+body > * {
```

все еще составной селектор
вместо body > * достаточно написано *
```suggestion
* {
```